docker-compose-dev.yaml

```diff
@@ -36,6 +36,7 @@ services:
     command: gunicorn main.wsgi:application --bind 0.0.0.0:8000 --workers=4
     ports:
       - 8000:8000
+    user: "root"
```

Comment:
@paaragon can we add a comment about why are we using the root here? I think it's interesting to annotate that this root is for the `watch`.